ui/src/App.tsx

```diff
@@ -33,24 +34,24 @@ const App: FC = () => {
         <Suspense fallback={<Loader />}>
           <Routes>
             <Route
-              path="/"
+              path={basePath}
```

Comment:
Instead of manipulating the routes, links and navigate calls you can set `basename` on the `<Router>` in index.tsx which will automatically make all other URLs relative to the base (see the last paragraph on this page: https://reactrouter.com/en/main/router-components/router).

Reply:
Good idea, using the react router base path now.